deploy/dotcom/prod/Dockerfile

```diff
@@ -46,6 +46,7 @@ RUN apk add curl
 RUN apk add libssl3
 
 COPY --from=builder /home/_build /home/_build
+COPY --from=builder /tmp/.functions /tmp/.functions
```

Comment:
📓 We have to build the list of functions at compile time. We use the file in the `tmp` directory so that it survives the image build process.